suite-native/navigation/src/hooks/useKeyboardFakeView.tsx

```diff
@@ -0,0 +1,19 @@
+import { useKeyboardHandler } from 'react-native-keyboard-controller';
+import Animated, { useAnimatedStyle, useSharedValue } from 'react-native-reanimated';
+
+export const useKeyboardFakeView = () => {
+    const height = useSharedValue(0);
+
+    useKeyboardHandler({
+        onMove: event => {
+            'worklet';
+            height.value = Math.max(event.height, 0);
+        },
+    });
+
+    const keyboardFakeViewStyle = useAnimatedStyle(() => ({
+        height: height.value,
+    }));
```

Comment:
nit: you can simplify this hook by using `useAnimatedKeyboard` from reanimated lib.
```suggestion
    const keyboard = useAnimatedKeyboard();

    const keyboardFakeViewStyle = useAnimatedStyle(() => ({
        height: keyboard.height.value,
    }));
```

Reply:
Also this should work (it is using the same hook under the hood):
```
import { useKeyboardAnimation } from "react-native-keyboard-controller";
...
const { height } = useKeyboardAnimation();



```

Reply:
It seems that `useAnimatedKeyboard` works fine. 👍